addons/dialogue_manager/DialogueManager.cs

```diff
@@ -122,13 +121,16 @@ public static async Task<GodotObject> GetSingleton()
       return new DialogueLine((RefCounted)result[0]);
     }
 
+    public static async void Mutate(Dictionary mutation, bool isInlineMutation, Array<Variant>? extraGameStates = null) {
```

Comment:
These arguments should match the order and defaults of the GDScript version.

Reply:
Never mind - there were a few other things that needed fixing too so I've just done them.